models/classes/runner/QtiRunnerService.php

```diff
@@ -975,10 +978,20 @@ public function displayFeedbacks(RunnerServiceContext $context)
         /* @var TestSession $session */
         $session = $context->getTestSession();
 
-        return $session->getCurrentSubmissionMode() !== SubmissionMode::SIMULTANEOUS
-            && $session->getCurrentAssessmentItemSession()->getItemSessionControl()->mustShowFeedback();
-    }
+        if ($session->getCurrentSubmissionMode() === SubmissionMode::SIMULTANEOUS) {
+            return false;
+        }
+
+        $data = $context->getTestMeta();
+        $testCompilationVersion = $data[CompilationDataService::COMPILATION_VERSION] ?? 0;
+
+        if ($testCompilationVersion > 0) {
+            return $session->getCurrentAssessmentItemSession()->getItemSessionControl()->mustShowFeedback();
+        }
```

Comment:
Thanks for the thoughts JS. Let me try to explain the idea to see if we are on the same page: 

- For new delivery publications, version will be '1', so we respect the `showFeedback` checkbox (as business expect to work). 
- For old publications, we will keep the previous behavior (show always). 
- For cases where we want to force, we can use the feature flag. 

My initial commit was only relying on feature flag, but I introduced the version to handle other cases. I will ask business validation on this too.

I agree it can cause confusion, and I am ok on removing the `version` too. I will just confirm with business :)

Reply:
After discussion with business we still keep the feature flag, but only available for old deliveries. So we need to activate this feature flag for the customer.